PSReadLine/Completion.cs

```diff
@@ -467,9 +554,31 @@ private void PossibleCompletionsImpl(CommandCompletion completions, bool menuSel
                     {
                         selectedItem = Math.Max(selectedItem - 1, 0);
                     }
+                    else if (nextKey == Keys.PageDown)
+                    {
+                        selectedItem = Math.Min(selectedItem + displayRows - (selectedItem % displayRows) - 1, matches.Count - 1);
+                    }
+                    else if (nextKey == Keys.PageUp)
+                    {
+                        selectedItem = Math.Max(selectedItem - (selectedItem % displayRows), 0);
+                    }
                     else if (nextKey == Keys.Tab)
                     {
-                        selectedItem = (selectedItem + 1) % matches.Count;
+                        string unAmbiguousText = GetUnambiguousPrefix(matches, true, out ambiguous);
```

Comment:
There are couple more `true` and `false` parameters that should use named parameters like is done elsewhere.

Could you also add comments to this code? I'm not sure what's going on, and whatever the intent, I'm confused because I don't see similar code in the `Shift+Tab` case when if feels like the inverse probably needs similar treatment.

I am wondering if you're trying to fix a long-standing bug where I show two identical items in the menu but they have different replacement text. If that's the case, this doesn't feel like the right fix.